### ISSUE TITLE
[Accessibility] fd-shellbar with fdPopoverTrigger add not-allowed aria-expanded attribute

### ISSUE THREAD

**Issue by itsnode** [SAP/fundamental-ngx#10968]
#### Is this a bug, enhancement, or feature request?
Bug
#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
A simple shellbar component which 

#### Please provide relevant source code (if applicable).
code sniipets

```
<fd-shellbar-actions [closePopoverOnSelect]="true">
    <fd-shellbar-user-menu
      #shellbarUserMenu  
      [fdPopoverTrigger]="userMenuPopover">
    </fd-shellbar-user-menu>
  </fd-shellbar-actions>
  <fd-popover
    #userMenuPopover >
    <fd-popover-body> </fd-popover-body>
</fd-popover>
```
#### Please provide stackblitz example(s).
https://stackblitz.com/edit/angular-usdxnt-bkrewm?file=src%2Fapp%2Flist-example.component.html
Please analyze the fd-shellbar-actions using https://chromewebstore.google.com/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm

you will see the error messages below.

<img width="1792" alt="Screenshot 2023-11-15 at 5 38 22 PM" src="https://github.com/SAP/fundamental-ngx/assets/48496287/62dfe548-9905-483c-b32d-6b0cf50c4516">
<img width="1792" alt="Screenshot 2023-11-15 at 5 37 32 PM" src="https://github.com/SAP/fundamental-ngx/assets/48496287/f976776c-bd99-4e6e-b57e-9d5abbeb4bcb">

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Got the Accessibility ticket from Sap Central Accessibility team. https://product-jira.ariba.com/browse/SA-44427

**Comment by itsnode** [comment]
@skaquib123 Noted that I notice fdMenuTrigger has         '[attr.aria-expanded]': 'isMenuOpen || null', but fdPopoverTrigger(which in our use case) does not have any to null out, maybe that could be a cause.
`
import {
    AfterContentInit,
    Directive,
    ElementRef,
    HostListener,
    Input,
    OnDestroy,
    Optional,
    Self,
    ViewContainerRef
} from '@angular/core';
import { ConnectedPosition, Overlay, OverlayConfig, OverlayRef } from '@angular/cdk/overlay';
import { TemplatePortal } from '@angular/cdk/portal';
import { ENTER, LEFT_ARROW, RIGHT_ARROW, SPACE } from '@angular/cdk/keycodes';
import { fromEvent, Subscription } from 'rxjs';
import { filter, take } from 'rxjs/operators';

import { KeyUtil } from '@fundamental-ngx/cdk/utils';
import { MenuCloseMethod, MenuComponent } from './menu.component';
import { MenuItemComponent } from './menu-item.component';

@Directive({
    selector: `[fdpMenuTriggerFor]`,
    host: {
        'aria-haspopup': 'menu',
        '[attr.aria-expanded]': 'isMenuOpen || null',
        '[attr.aria-controls]': 'isMenuOpen ? menu.menuId : null'
    }
})`

**Comment by droshev** [comment]
@itsnode is this still valid?

**Comment by droshev** [comment]
no activities => closing the issue
